Tasks/NuGetCommandV2/UbuntuDetectionHelper.ts

```diff
@@ -0,0 +1,77 @@
+import * as os from 'os';
+import * as fs from 'fs';
+import * as tl from "azure-pipelines-task-lib/task";
+
+export function detectUnsupportedUbuntuVersion(): boolean {
+    const platform = os.platform();
+    if (platform === 'linux') {
+        const lbsContents = _readLinuxVersionFile();
+        const distribution = _parseLinuxDistribution(lbsContents);
+        if (distribution === 'Ubuntu') {
+            const version = parseFloat(_parseUbuntuVersion(lbsContents));
+            if (version >= 22.04) {
```

Comment:
I was mistaken I apologize. I'm not sure where I got 22.04 from. 24.04 should be the proper cutoff (though as long as we check for mono as well, it shouldn't be a big deal)